src/OrleansPSUtils/OrleansPSUtils.2017.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansPSUtils</PackageId>
```

Comment:
That's what I did, updated to 5.x which is the first xplat version. We should do that for  vNext and master too.

On the other hand I'd like to have the PSUtils installable from nuget, so we need a nuget package too. I started that work in the 2017 csprojs, but since I'm not a PS guru I don't know the exact packaging needs.